analyser/apkAnalyser.go

```diff
@@ -0,0 +1,77 @@
+package analyser
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+)
+
+//apkAnalyser is an IAnalyser implementation for Android platform
+type apkAnalyser struct {
+	filePath *string
+}
+
+//PrintAppInfo fetches and prints information about the Android app package
+func (analyser apkAnalyser) PrintAppInfo() {
+	appInfo := analyser.getAppInfo()
+	fmt.Printf("Recognised app info:\n "+
+		"\tPackage Name: %+v\n "+
+		"\tVersion name: %+v\n "+
+		"\tVersion code: %+v\n "+
+		"\tApp icon file path: %+v\n",
+		appInfo.AppId,
+		appInfo.Version,
+		appInfo.Build,
+		appInfo.IconPath)
+}
+
+//getAppInfo returns populated AppInfo
+func (analyser apkAnalyser) getAppInfo() *AppInfo {
+	buildToolsVersion := getBuildToolsVersion()
+	buildToolPath := "/build-tools/" + buildToolsVersion
+	androidSDKPath := os.ExpandEnv("$ANDROID_HOME")
```

Comment:
`ANDROID_HOME` is also expanded in `getBuildToolsVersion`, getBuildToolsVersion could return the full build tool path, this way, you could remove this line